Helm: run under Restricted pod security policy (#3007)
grafana/mimir: operations/helm/charts/mimir-distributed/README.md

have varying resource requirements. Please make sure that you have sufficient
resources (CPU/memory) available in your cluster before installing Grafana Mimir Helm Chart.

> For a more detailed guide on setting up Grafana Mimir refer to [Getting started with Grafana Mimir using the Helm chart](https://grafana.com/docs/mimir/latest/operators-guide/deploying-grafana-mimir/getting-started-helm-charts/).

Reviewer: ```suggestion
For details about setting up Grafana Mimir, refer to [Get started with Grafana Mimir using the Helm chart](https://grafana.com/docs/mimir/latest/operators-guide/deploy-grafana-mimir/get-started-helm-charts/).
```
Some of these changes are likely out of scope for this particular PR. Please take at least the first part of the suggestion.
